app/models/mixins/virtual_total_mixin.rb

```diff
@@ -16,7 +16,7 @@ module ClassMethods
     #
     def virtual_total(name, relation)
       define_method(name) do
-        send(relation).count
+        send(relation).try(:size)
```

Comment:
@kbrock hello, as a followup to our PR https://github.com/ManageIQ/manageiq/pull/8393

I had the before total_vms as vms.try(:size), since vms can be a delegation instead of a relation and the object we are delegating to doesn't have to be there, it can result to bil, instead of arel.

Also is it better to use size, in the case the relation is already loaded?